packages/loot-core/src/types/server-handlers.d.ts

```diff
@@ -118,7 +119,7 @@ export interface ServerHandlers {
 
   'payees-check-orphaned': (arg: { ids }) => Promise<unknown>;
 
-  'payees-get-rules': (arg: { id }) => Promise<unknown>;
+  'payees-get-rules': (arg: { id }) => Promise<RuleEntity[]>;
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏I know this is not new code, but if we're touching it.. might as well improve it even further :)

```suggestion
  'payees-get-rules': (arg: { id: string }) => Promise<RuleEntity[]>;
```